src/main/java/it/gov/pagopa/pu/classification/service/BalanceService.java

```diff
@@ -1,27 +1,100 @@
 package it.gov.pagopa.pu.classification.service;
 
+import it.gov.pagopa.pu.classification.enums.AssessmentsRegistryStatus;
 import it.gov.pagopa.pu.classification.exception.custom.InvalidValueException;
+import it.gov.pagopa.pu.classification.model.AssessmentsRegistry;
+import it.gov.pagopa.pu.classification.repository.AssessmentsRegistryRepository;
+import it.veneto.regione.schemas._2012.pagamenti.ente.bilanciodefault.CtAccertamentoDefault;
+import it.veneto.regione.schemas._2012.pagamenti.ente.bilanciodefault.CtBilancioDefault;
+import it.veneto.regione.schemas._2012.pagamenti.ente.bilanciodefault.CtCapitoloDefault;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
 import org.springframework.stereotype.Service;
 
+import java.time.LocalDate;
+import java.util.Set;
+
 @Service
 @Slf4j
 public class BalanceService {
 
-  private final BalanceUnmashallerService balanceUnmashallerService;
+  private static final String OPERATING_YEAR = String.valueOf(LocalDate.now().getYear());
```

Comment:
this is wrong! you cannot store it as costant value, otherwise you will work with an invalid value if the new year arrives and the application is not restarted